### PR TITLE
Add rgen test

### PR DESCRIPTION
**Pull request by duongphammmm**

### CODE REVIEW

**Review by wardetu — APPROVED** [review]
LGTM!

**Review by helloImHai — CHANGES_REQUESTED** [review]
Since you're already doing the tests for rgen can you also add the test for rgen parser?

File: src/main/java/seedu/address/logic/parser/GenerateResumeCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the GenerateResumeCommand
+     * and returns a GenerateResumeCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public GenerateResumeCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME);
+        if (argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(GenerateResumeCommand.MESSAGE_USAGE);
```
Comment (helloImHai) [inline src/main/java/seedu/address/logic/parser/GenerateResumeCommandParser.java]: I think the new convention is that we'll wrap this around a "Invalid Command Format!\n...." can look at my PR to see? Not all but whenever we call the XYZ.Command.MESSAGE_USAGE it's quite clear

**Review by helloImHai — APPROVED** [review]
Nice one bro 🥇Except for that 1 convention change? I'll approve, real quick change then can self-merge!

**Pull request merged**